### ISSUE TITLE
"Emoji" plugin - A simple request that would mean the world

### ISSUE THREAD

**Issue by iHateStairs** [runelite/runelite#13142]
I've been in the same clan chat now since Oldschool RuneScape was released. It may seem odd, but we're a clan made up entirely of paraplegic/wheelchair-bound players. We're a pretty closed off group (I'm sure you could imagine the trolling that comes with our situation) but we've not had less than 20 active players at a time, all wheelchair-bound.

Recently we all collectively decided to keep the "Emoji" plugin turned on to spice up the clan chat. We quickly noticed on [the plugin's Wiki page](https://github.com/runelite/runelite/wiki/Emojis) that the wheelchair emoji (♿) wasn't one of the emojis available.

We've considered just being content with pretending there _is_ a wheelchair emoji by typing "o:)o", though we thought it would be a good idea to at _least_ suggest it to be added, and while yes a very minor addition (and probably more annoying to do than it is difficult) it would mean the absolute WORLD to us if you could add this emoji so we could use it to empower each other.

Please if you could, find it in the bottom of your hearts to add this emoji. We would be **FOREVER** grateful!!

**Comment by ErinTales** [comment]
This emoji is used solely as an insult, it's ableist and promotes toxicity.
Would very much prefer if it was not added.

**Comment by aramzan** [comment]
> This emoji is used solely as an insult, it's ableist and promotes toxicity.
> Would very much prefer if it was not added.

I disagree. I've never encountered "o:)o" as an insult and I doubt it would become popular with the addition of an emoji. People with emojis disabled would have no idea what's going on and supporting a disabled community would send a good message for the promotion of accessibility.

**Comment by MarbleTurtle** [comment]
> > This emoji is used solely as an insult, it's ableist and promotes toxicity.
> > Would very much prefer if it was not added.
> 
> I disagree. I've never encountered "o:)o" as an insult and I doubt it would become popular with the addition of an emoji. People with emojis disabled would have no idea what's going on and supporting a disabled community would send a good message for the promotion of accessibility.

o:)o may not be an insult you've ever seen but disabled or handicapped certainly is, and while yes people without the emojis on wouldn't know what it means its far more likely that people DO have it on. Regardless, it's known that the person who posted this is a fake as the picture they used is from a new article so it's safe to say that they don't have the best intentions with the emoji.
